command/results_xunit.go

```diff
@@ -136,6 +138,18 @@ func (c *xunitResults) parseAndUploadResults(ctx context.Context, conf *model.Ta
 
 		// go through all the tests
 		for _, suite := range testSuites {
+			if len(suite.TestCases) == 0 && suite.Error != nil {
+				// if no test cases but an error, generate a default test case
+				tc := testCase{
+					Name:  suite.Name,
+					Time:  suite.Time,
+					Error: suite.Error,
+				}
+				if tc.Name == "" {
+					tc.Name = autoGenTestName
```

Comment:
can we call this "`Unamed Test-<idx>`" using the sequence number from the submitted tests?